public/app/features/explore/TraceView/components/common/Divider.tsx

```diff
@@ -26,7 +26,8 @@ const getStyles = (theme: GrafanaTheme2) => {
       display: block;
       height: 1px;
       width: 100%;
-      margin: 24px 0;
+      margin-top: ${theme.spacing(2)};
+      margin-bottom: ${theme.spacing(0.5)};
```

Comment:
Will also add theme spacing to other places to ensure we have consistent spacing